app/forms/hyrax/rcr_form.rb

```diff
@@ -5,5 +5,16 @@ class RcrForm < Hyrax::Forms::WorkForm
     self.model_class = ::Rcr
     self.terms += Tufts::Terms.shared_terms
     self.required_fields = [:title, :displays_in]
+    self.field_metadata_service = Tufts::MetadataService
+
+    def self.model_attributes(_)
+      attrs = super
+      attrs[:title] = Array(attrs[:title]) if attrs[:title]
+      attrs
+    end
+
+    def title
+      super.first || ""
```

Comment:
Can you say something about what this is doing. My fast read is that it is going to assign empty strings as titles when none exist? Or is this just a null title case for form objects, that doesn't impact settings? (in the latter case :+1:).

Also, let's think about how to dedup some of the code in this PR, since it needs to wait until monday anyway.